app/controllers/membership_applications_controller.rb

```diff
@@ -21,11 +21,6 @@ def index
   end
 
 
-  def show
-    @categories = @membership_application.business_categories
-  end
```

Comment:
`def show ....` should not have been deleted.  I think that was just an automatic `merge` that was overzealous.  (The same kind of merge resolution happened on my local dev machine:  stuff was deleted by the overzealous merge.) I'm fixing that now. 

(I'm looking at the `destroy...` _do we need to get the categories?_  stuff)